pkg/component/shared/istio_test.go

```diff
@@ -151,25 +162,29 @@ func checkAdditionalIstioGateway(istioDeploy istio.Interface,
 		maxReplicas = ingressValues[0].MaxReplicas
 	} else {
 		zones = []string{*zone}
+
+		enforceSpreadAcrossHosts, err = ShouldEnforceSpreadAcrossHosts(context.TODO(), cl, []string{*zone})
```

Comment:
```suggestion=
		enforceSpreadAcrossHosts, err = ShouldEnforceSpreadAcrossHosts(context.Background(), cl, []string{*zone})
```

Reply:
Replaced all `context.TODO()` in the test file as suggested.